tests/actions.go

```diff
@@ -771,8 +771,12 @@ func (oa *operatorActions) DeployTidbCluster(info *TidbClusterConfig) error {
 		return fmt.Errorf("failed to create secret of cluster [%s]: %v", info.ClusterName, err)
 	}
 
-	cmd := fmt.Sprintf("helm install %s  --name %s --namespace %s --set-string %s --set %s",
-		oa.tidbClusterChartPath(info.OperatorTag), info.ClusterName, info.Namespace, info.TidbClusterHelmSetString(nil), info.TidbClusterHelmSetBoolean(nil))
+	cmd := fmt.Sprintf("helm install %s %s --namespace %s --set-string %s --set %s --set-file tikv.config=/etc/tikv.toml",
```

Comment:
The PR that upgrade from helm2 to helm3 was not cherry-picked to release-1.1, so we have to update the command to helm2 here.